src/Model/GalleryMediaCollectionInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Model;
+
+/**
+ * This a workaround to maintain BC within SonataMediaBundle v3.
+ * NEXT_MAJOR: remove this interface, move all methods into GalleryInterface
+ */
+interface GalleryMediaCollectionInterface
+{
+    /**
+     * @param GalleryHasMediaInterface $galleryHasMedia
+     */
```

Comment:
You can remove this docblock, we already get this piece of information from the docblock

Reply:
Did you mean we already get this information from the type hint? I just added it there to be consistent with code style in interfaces where there are dockblocks even if there is also a type hint.

Reply:
Yeah, I'm trying to remove them when I can but Sonata is huge.